app/src/main/res/values/strings.xml

```diff
@@ -1068,4 +1070,14 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_camera_type_dome">Dome</string>
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
+    <string name="no_search_results">The what now?</string>
+    <string name="quest_selection_hint_title">Too much to do?</string>
+    <string name="quest_selection_hint_message">"Contributing should primarily be fun!
+
+If you are overwhelmed by the number of quests, you can always fine-tune what quests are displayed to you and in what order in the settings.</string>
```

Comment:
```suggestion
If you are overwhelmed by the number of quests, you can always fine-tune which quests are displayed to you and in which order in the settings.</string>
```

CC @peternewman @smichel17 again.

Reply:
I think it should be "which quests… what order"
```suggestion
If you are overwhelmed by the number of quests, you can always fine-tune which quests are displayed to you and in what order in the settings.</string>
```